src/Apps/Artwork/Components/ArtworkImages.tsx

```diff
@@ -5,13 +5,20 @@ import { renderWithLoadProgress } from "Artsy/Relay/renderWithLoadProgress"
 import React from "react"
 import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
 import { ImageCarousel } from "./ImageCarousel"
+import { SaveButton } from "./SaveButton"
+import { ShareButton } from "./ShareButton"
 
 interface ArtworkImagesProps {
   artwork: ArtworkImages_artwork
 }
 
 const ArtworkImages: React.SFC<ArtworkImagesProps> = props => {
-  return <ImageCarousel images={props.artwork.images} />
+  return (
+    <ImageCarousel
+      images={props.artwork.images}
+      actions={[<SaveButton />, <ShareButton />]}
```

Comment:
This is totally fine but note that another (slightly more flexible) way of writing this would be to use a fragment:
```jsx
actions={
  <>
     <SaveButton />
     <ShareButton />
  </>
}
```
That way we're staying inside of more common JSX idioms.

Reply:
I wasn't aware of fragments, so I initially tried to do this by wrapping the buttons in a div tag. However, I decided to pass these buttons as a prop so that we could pass more than action buttons as children of an image carousel if it becomes necessary in the future.

Reply:
👍